### PR TITLE
Upgrade django to latest version

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**
LEARNER-1306

FYI: @edx/learner @edx/learner-spartans

### CODE REVIEW

File: ecommerce/settings/base.py [hunk 1]
```diff
@@ -155,7 +155,7 @@
 # SITE CONFIGURATION
 # Hosts/domain names that are valid for this site
 # See https://docs.djangoproject.com/en/1.5/ref/settings/#allowed-hosts
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['*']
```
Comment (bderusha) [inline ecommerce/settings/base.py]: do we override this in production?

**Review by bderusha — APPROVED** [review]
One question about the ALLOWED_HOSTS setting.  Just want to make sure this doesn't open us up to any security risk.  Otherwise looks good!

**Review by mjfrey — APPROVED** [review]
Looks good.

File: ecommerce/settings/base.py (hunk 1 above)
Comment (ahsan-ul-haq) [reply ecommerce/settings/base.py]: I don't think so. We can add it in the test settings and local settings and keep it `[]` here.

**Pull request merged**